### DISCUSSION TITLE
add documentation for collection.readByQuery  (collection.readMany is not wokring with any parameters))

### DISCUSSION THREAD

**Discussion by sambitraze** [directus/directus#10913] [Archive]
`const item = await collectionService.readByQuery({
		filters: {
			"crontill": {
				"_gte": "$NOW"
			}
		},
	});` 
	this is working 
	`const item = await collectionService.readmany({
		filters: {
			crontill: {
				_gte: "$NOW"
			}
		},
	});`
this is not working

**Comment by rijkvanzanten** [comment]
@sambitraze `readByQuery` reads items by a passed query (like you're doing). `readMany` reads by multiple primary keys, like `readMany([1, 5, 12])` and has an (optional) query object as the second parameter

**Reply by sambitraze** [reply]
Yes i found out the solution but the documentation is misleading as you can see in the screenshot below
![image](https://user-images.githubusercontent.com/23294656/148637010-03b24555-e604-47a4-aa5d-af513055a21a.png)
i think it should be updated

**Reply by azrikahar** [reply]
@sambitraze thank you for pointing that out. Updated it in #10943 👍
